### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by EsmeeIDEMS** [IDEMSInternational/parenting-app-ui#1010]
**Describe the bug**
Probably as a knock-on of a recent PR, the Options button on the Weekly Workshops page no longer becomes higher when the title text wraps over two lines. 

**App version**
v0.11.2 **+ some merged PRs**

**To Reproduce**
On the home screen, click Weekly Workshops tile to go to the Weekly Workshops overview page.

**Template Name**
 [weekly_workshops](https://docs.google.com/spreadsheets/d/1tmtdyeSYcqGH6joqZiA-wPtH0QH6td8hA3qGp62AcHs/edit#gid=1277645488)

**Screenshots and videos**
Screenshot from UAT, i.e. released app v0.11.2
![image](https://user-images.githubusercontent.com/74557272/132490323-1cb2cdd0-8c33-49f8-8036-23ef8746ad0e.png)

Screenshot from web app, i.e. current master branch
![image](https://user-images.githubusercontent.com/74557272/132490541-9f28cc9a-97e8-4aa9-a028-610e2cf3228b.png)

**Comment by EsmeeIDEMS** [comment]
I've marked this as High Priority, assuming that it's probably easier to find out which PR caused this unwanted side effect sooner rather than later.

**Comment by chrismclarke** [comment]
Makes sense to me, I'll quickly check on slack if someone can take this on
